### ISSUE TITLE
BUG: scipy.stats.multivariate_t distribution math documentation

### ISSUE THREAD

**Issue by alegresor** [scipy/scipy#14608]
### Reproducing Code Example

```python
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.multivariate_t.html#scipy.stats.multivariate_t

the math formula for the probability density function contains an extraneous exp. See the multivariate t-distribution wikipedia page at https://en.wikipedia.org/wiki/Multivariate_t-distribution
```

### Error message

```shell
No error message
```

### SciPy/NumPy/Python version information

1.7.1 documentation

**Comment by tirthasheshpatel** [comment]
That does seem like an error in documentation. Would you like to submit a pull request?

**Comment by alegresor** [comment]
> That does seem like an error in documentation. Thanks for reporting! Would you like to submit a pull request?

I have created a pull request at https://github.com/scipy/scipy/pull/14635

**Comment by tirthasheshpatel** [comment]
Closed in #14635
